common/app/routes/Settings/components/Cert-Settings.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import _ from 'lodash';
+import { values as _values, isString, findIndex } from 'lodash';
```

Comment:
Why are you breaking out these values instead of just using the underscore value?

Reply:
Even with the babel plugin, we are still bundling the whole of `lodash`. So I thought if I de-structure as I go, we will eventually only bundle what is required.

Reply:
Are you sure about that? That is the whole point of the babel plugin.